File: 백준/Bronze/2562. 최댓값/최댓값.py
numbers = [int(input()) for _ in range(9)]
max_number = -1
max_index = -1
for i in range(9):
    if numbers[i] > max_number:
        max_number = numbers[i]
        max_index = i + 1
        
print(max_number)
print(max_index)

File: 백준/Bronze/2562. 최댓값/최댓값.kt
fun main() {
    // 9개의 요소를 가진 리스트를 생성하고, 각 요소를 입력받아 정수로 변환하여 저장
    val numbers = List(9) { readLine()!!.toInt() }    

    var max_number = -1
    var max_index = -1

    for (i in 0 until 9) {
        if (numbers[i] > max_number) {
            max_number = numbers[i]
            max_index = i
        }
    }

    println(max_number)
    println(max_index + 1)
}